docs/tutorial/index.rst

```diff
@@ -215,10 +215,10 @@ old schedule::
 
 
     >>> func = objective_functions.number_of_changes
-    >>> schedule = scheduler.schedule(events, slots, objective_function=func, original_schedule=schedule)
+    >>> optimal_schedule = scheduler.schedule(events, slots, objective_function=func, original_schedule=schedule)
 
-    >>> schedule = sorted(schedule, key=lambda item: item.slot.starts_at)
-    >>> for item in schedule:
+    >>> optimal_schedule = sorted(optimal_schedule, key=lambda item: item.slot.starts_at)
```

Comment:
Both are optimal just according to different objective functions so perhaps call this one `similar_schedule` ?